<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>965</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Score Capture</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_5">
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter_2">
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
      <property name="opaqueResize">
       <bool>true</bool>
      </property>
      <property name="childrenCollapsible">
       <bool>false</bool>
      </property>
      <widget class="QFrame" name="frame_sidebar">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>130</width>
         <height>0</height>
        </size>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label_logo">
          <property name="font">
           <font>
            <pointsize>22</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Score
Capture</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_locate">
          <property name="text">
           <string>定位</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_preview">
          <property name="text">
           <string>预览</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_capture">
          <property name="text">
           <string>开始截图</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_stitch">
          <property name="text">
           <string>拼接</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_reclip">
          <property name="text">
           <string>重分割</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Orientation::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_instruction">
          <property name="text">
           <string>打开说明文档</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_author">
          <property name="text">
           <string>Carrot shreds</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QSplitter" name="splitter">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
       <property name="childrenCollapsible">
        <bool>false</bool>
       </property>
       <widget class="QWidget" name="layoutWidget">
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <layout class="QGridLayout" name="gridLayout">
           <item row="1" column="0">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>存储路径：</string>
             </property>
            </widget>
           </item>
           <item row="2" column="5">
            <widget class="QComboBox" name="comboBox_log_level">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>90</width>
               <height>0</height>
              </size>
             </property>
             <item>
              <property name="text">
               <string>DEBUG</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>INFO</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>SUCCESS</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>WARNING</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>ERROR</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="2" column="7">
            <widget class="QComboBox" name="comboBox_save_format">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>55</width>
               <height>0</height>
              </size>
             </property>
             <item>
              <property name="text">
               <string>.jpg</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>.png</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="lineEdit_score_title">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
           <item row="1" column="6" colspan="2">
            <widget class="QCheckBox" name="checkBox_auto_manage_config">
             <property name="text">
              <string>自动保存/加载配置</string>
             </property>
            </widget>
           </item>
           <item row="2" column="4">
            <widget class="QLabel" name="label_12">
             <property name="text">
              <string>日志等级：</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="2" column="2">
            <widget class="QPushButton" name="pushButton_select_folder">
             <property name="text">
              <string>选择文件夹</string>
             </property>
            </widget>
           </item>
           <item row="2" column="6">
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>存储格式：</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>曲谱标题：</string>
             </property>
            </widget>
           </item>
           <item row="1" column="5">
            <widget class="QPushButton" name="pushButton_select_path">
             <property name="text">
              <string>浏览目录</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1" colspan="4">
            <widget class="QLineEdit" name="lineEdit_save_path"/>
           </item>
           <item row="2" column="3">
            <widget class="QPushButton" name="pushButton_open_folder">
             <property name="text">
              <string>打开目录</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0" colspan="6">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>全局设置</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignmentFlag::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="0" column="6" colspan="2">
            <widget class="QCheckBox" name="checkBox_always_on_top">
             <property name="text">
              <string>窗口始终靠前</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="0" colspan="2">
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="1" column="3">
            <widget class="QDoubleSpinBox" name="doubleSpinBox_compare_threshold">
             <property name="maximum">
              <double>65535.000000000000000</double>
             </property>
             <property name="singleStep">
              <double>0.100000000000000</double>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QCheckBox" name="checkBox_reverse_image">
             <property name="text">
              <string>对图片进行反相</string>
             </property>
            </widget>
           </item>
           <item row="2" column="4">
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>宽:</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="0" column="0" colspan="4">
            <widget class="QLabel" name="label_15">
             <property name="text">
              <string>截图设置</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignmentFlag::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_14">
             <property name="layoutDirection">
              <enum>Qt::LayoutDirection::LeftToRight</enum>
             </property>
             <property name="text">
              <string>图片比较算法：</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="1" column="6">
            <widget class="QLabel" name="label_11">
             <property name="text">
              <string>Y:</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="1" column="5">
            <widget class="QSpinBox" name="spinBox_region_x">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>80</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximum">
              <number>2000</number>
             </property>
            </widget>
           </item>
           <item row="1" column="4">
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>X:</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="0" column="4" colspan="4">
            <widget class="QLabel" name="label_10">
             <property name="text">
              <string>定位设置</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignmentFlag::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="2" column="7">
            <widget class="QSpinBox" name="spinBox_region_height">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>80</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximum">
              <number>2000</number>
             </property>
            </widget>
           </item>
           <item row="1" column="7">
            <widget class="QSpinBox" name="spinBox_region_y">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>80</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximum">
              <number>2000</number>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QLabel" name="label_16">
             <property name="text">
              <string>截至差异值：</string>
             </property>
            </widget>
           </item>
           <item row="2" column="3">
            <widget class="QDoubleSpinBox" name="doubleSpinBox_capture_delay">
             <property name="singleStep">
              <double>0.100000000000000</double>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QComboBox" name="comboBox_compare_method">
             <item>
              <property name="text">
               <string>SSIM</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>MSE</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="3" column="0" colspan="8">
            <layout class="QGridLayout" name="gridLayout_6">
             <item row="1" column="2">
              <widget class="QLabel" name="label_18">
               <property name="layoutDirection">
                <enum>Qt::LayoutDirection::LeftToRight</enum>
               </property>
               <property name="text">
                <string>总小节数：</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QSpinBox" name="spinBox_horizontal_lines_num"/>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_17">
               <property name="layoutDirection">
                <enum>Qt::LayoutDirection::LeftToRight</enum>
               </property>
               <property name="text">
                <string>水平线数量：</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item row="1" column="3">
              <widget class="QSpinBox" name="spinBox_bar_lines_num"/>
             </item>
             <item row="0" column="0" colspan="4">
              <widget class="QLabel" name="label_24">
               <property name="text">
                <string>图像识别设置</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignmentFlag::AlignCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="2" column="5">
            <widget class="QSpinBox" name="spinBox_region_width">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>80</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximum">
              <number>2000</number>
             </property>
            </widget>
           </item>
           <item row="2" column="2">
            <widget class="QLabel" name="label_13">
             <property name="text">
              <string>截图间隔(秒/s)：</string>
             </property>
            </widget>
           </item>
           <item row="2" column="6">
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>高:</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QCheckBox" name="checkBox_keep_last">
             <property name="text">
              <string>保留最后一组截图</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QTextEdit" name="textEdit_console"/>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>965</width>
     <height>33</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
